dask_ml/_compat.py

```diff
@@ -29,6 +29,12 @@
 PANDAS_1_2_0 = PANDAS_VERSION > packaging.version.parse("1.2.0")
 WINDOWS = os.name == "nt"
 
+# 'log_loss' is preferred as of scikit-learn 1.1
+if SK_VERSION >= packaging.version.parse("1.1"):
```

Comment:
Wonder if we should set the result of this conditional to a variable (i.e. `SK_1_1`) if we intend to use it elsewhere in the codebase?

Reply:
+1 -- let's add an `SKLEARN_1_1_X` variable a lines above with the other similar variables (e.g. `PANDAS_1_2_0`). Also, for these version compatibility variables it helps to use the full X.Y.Z version -- that way it's more clear when we can remove related compatibility code